Dialog/DoorConfigDialog.cpp

```diff
@@ -282,6 +284,15 @@ void DoorConfigDialog::RenderGraphicsView_DestinationDoor(int doorIDinRoom)
     ui->GraphicsView_DestinationDoor->setScene(scene);
     ui->GraphicsView_DestinationDoor->setAlignment(Qt::AlignTop | Qt::AlignLeft);
 
+    if(tmpDestinationRoom->GetRoomID() == tmpCurrentRoom->GetRoomID())
```

Comment:
Which user action should cause WL4Editor to go into this if-statement?

Reply:
you can set a Door's destination door to be itself to refresh the room, and there is an edge case when you update the door size and position properties in the idalog but you point its destination to be itself, or its destination is inside the same room, then the destination door need an update, they are 2 room copies with different door data copies, so need to update them every time.